vcx/libvcx/src/credential.rs

```diff
@@ -70,12 +66,7 @@ pub struct Credential {
     credential_offer: Option<CredentialOffer>,
     msg_uid: Option<String>,
     // the following 6 are pulled from the connection object
-    agent_did: Option<String>,
```

Comment:
It breaks compatibility with previously serilized objects. I think we should add a new version of object.
Now the sequence of steps: `deserilize` -> `update_state` will fail.